flow/core/kernel/vehicle/traci.py

```diff
@@ -113,6 +124,13 @@ def initialize(self, vehicles):
                 self.__vehicles[veh_id] = dict()
                 self.__vehicles[veh_id]['type'] = typ['veh_id']
                 self.__vehicles[veh_id]['initial_speed'] = typ['initial_speed']
+                # self.energy.initialize(veh_id)
+                # energy.intialize(veh_id)
+                self.__vehicles[veh_id]['SOC'] = float(random.randrange(8, 92))/100 #move to experiment code
+                # # Assume level of fuel is 10-100% of capacity
+                # # Tacoma has 21.1 gallon capacity = 79.87219 L
+                # # Use density of gasoline = 748.9 g/L (needs to convert b/c fc is in g/s)
+                self.__vehicles[veh_id]['fuel'] = float(random.randrange(598163, 5981628))/100
```

Comment:
i think we may want to avoid using anything random here because everything else here is deterministic -- but i'm not sure yet what's a better alternative.

Reply:
i think the instantaneous power is not a function of fuel, right? so we don't need to track it (do we?).

for SOC, maybe we should just pick an estimated value, like 0.5, but track it inside the `ToyotaPriusModel`, not out here.